harvest/storage/base_storage.py

```diff
@@ -166,28 +166,28 @@ def load(
                     return data
             return None
 
-        dt_interval = interval_to_timedelta(interval)
-
-        self.storage_lock.acquire()
-        if interval not in self.storage[symbol]:
```

Comment:
Disabling this feature, as the `interval_to_timedelta` made it difficult to handle cases where "special" intervals, like `-9`, were needed. These are intervals used by `BackTester` to create data needed for simulations.